helm/provisioner/values.yaml

```diff
@@ -106,7 +106,10 @@ daemonset:
   #
   # Defines Provisioner's image name including container registry.
   #
-  image: quay.io/external_storage/local-volume-provisioner:v2.3.4
+  # Since 2.4.0 we push multi-arch image to gcr.io/k8s-staging-sig-storage/local-volume-provisioner.
+  # Use it when you deploy in non-amd64 environment.
+  #
+  image: quay.io/external_storage/local-volume-provisioner:v2.4.0
```

Comment:
Should we use k8s.gcr.io/sig-storage registry instead? (We'll need to promote the staging image once we tag)

Reply:
pushed a new commit https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/pull/229/commits/46da21477d8590007ad1b9bd102e245b83bf1d9b PTAL